styles.css

```diff
@@ -972,7 +968,123 @@ footer {
   display: none;
 }
 
-#get-resume a:hover {
+#get-resume:hover {
   background-color: #6070ff;
   color: rgba(255, 255, 255, 1);
 }
+
+#pj1 {
+  position: absolute;
+  height: 71.3rem;
+  top: 10px;
+  z-index: 10;
+  padding: 0 0 1.6rem 1.6rem;
+  margin: 1.6rem;
+  border: 1px solid rgba(223, 225, 230, 1);
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+#pj1-h2 {
+  font-family: "poppins", sans-serif;
+  font-size: 3.2rem;
+  font-weight: 700;
+  line-height: 4.4rem;
+  letter-spacing: 0;
+  text-align: left;
+  height: 3.6rem;
+  margin-top: 1.2rem;
+  margin-bottom: -0.2rem;
+  color: rgba(23, 43, 77, 1);
+}
```

Comment:
Every time the popup opens, it is displayed at the top of the web page. Consider making it absolute with the current position of the card for which you are generating the popup rather than making it absolute relative to whole body.

Reply:
![image](https://user-images.githubusercontent.com/104029494/175763696-5f072b57-7472-4ab9-8e44-b2946ed94744.png)
As you can see from the figma template, the popup is appearing on the top of the page, meaning all the buttons on click should display the popup on the top of the page. If they were to be absolute to the relative position of the card, then the background visible would have been a card. I can't effect those changes as they contradict the Figma.